playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/post.yml

```diff
@@ -52,7 +52,7 @@
   - name: Update registry image to current version
     when: _default_registry.rc == 0
     command: >
-      {{ oc_cmd }} patch dc/docker-registry -p
+      {{ oc_cmd }} patch dc/docker-registry -p -n default
```

Comment:
Does openshift-ansible execute oc project default to enforce the correct context? What happens, if I execute ``oc project dev`` before I execute the ansible?